css/sponsors.css

```diff
@@ -160,3 +172,6 @@ h1 {
     font-weight: 600;
     color: #666;
 }
+h1.col-md-12{
+    padding-bottom: 50px;
+}
```

Comment:
Oh, thanks for the heads up I did not know that. Any reason why? Also could you review the other merge request? I addressed all the concerns here but there may be more minor things